tracker/view/advisory.py

```diff
@@ -1,7 +1,10 @@
 from collections import OrderedDict
 from re import match
 
+import pytz
```

Comment:
mostly we try to just import what we need, can we changed this to only import `UTC`?